examples/controls/control.css

```diff
@@ -0,0 +1,12 @@
+#controlDiv {
+      position:absolute;
+      top:10px;
+      right:10px;
+      width:150px;
+      height:150px;
+      background-color: white;
+      border-style: solid;
+      border-color: black;
+      border-width: 1px;
+      box-shadow: 1px 1px 10px #555;
+}
```

Comment:
I actually suggest:

```css
position: absolute;
top: 1rem;
right: 1rem;
width: 20rem;
box-shadow: 1px 1px 10px #555;
font: 11px Lucida Grande,sans-serif;
color: #eee;
padding: 0.5rem;
```

And separating the fields with `<br>` instead of `;` in your callbacks (in `pickposition_controls`).

Result:

![screen shot 2017-01-16 at 15 22 27](https://cloud.githubusercontent.com/assets/1576598/21986416/a69406fa-dbff-11e6-989c-6bc3355eccfa.png)